docs/source/model_design/vivarium_features/other_causes_ylds/index.rst

```diff
@@ -81,16 +91,25 @@ sequela prevalence.
 
 .. note::
 
-  The COMO process in GBD assumes total independence between health states and 
-  does not account for any correlation that may exist between related health 
-  states (beyond correlation induced by location, age, and sex, which is 
-  accounted for through its stratified adjustments).
+  The COMO adjustment is run and maintained by IHME central computation. `The 
+  repository can be found here (as on May, 2023) <https://stash.ihme.washington.edu/projects/CCGMAC/repos/como/browse>`_.
+
+  According to correspondence with a central computation representative: the COMO
+  adjustment ensures that no simulant may simultaneously possess mutually exclusive
+  sequelae (mild AND moderate diarrhea, for instance), nor may they simultaneously
+  possess mutually exclusive impairments (such as mild AND moderate anemia). A 
+  simulant may, however, simultaneously possess more than one sequelae that correspond
+  to the same healthstate. 
```

Comment:
This is precise but a bit difficult to follow if you are not familiar with healthstates. My understanding after the meeting yesterday is that if a person has two illnesses that both have the same general symptoms, it's assumed that they have those symptoms roughly twice as intensely (I know it isn't exactly twice, since they combine multiplicatively).